packages/nucondb/package.py

```diff
@@ -9,15 +9,20 @@
 class Nucondb(MakefilePackage):
     """Data handling client code for intensity frontier experiments"""
 
-    homepage = "https://cdcvs.fnal.gov/redmine/projects/nucondb"
-    url = "https://cdcvs.fnal.gov/cgi-bin/git_archive.cgi/cvs/projects/nucondb.v2_2_10.tbz2"
-    git = "https://cdcvs.fnal.gov/projects/ifdhc-nucondb"
-
-    version("2.6.1", sha256="fbf59259e8cd75334b4ad17d3da46519d35c5c81ccf55b8df2d19f664fe016b9")
-    version("2.5.23", sha256="30399133b64548affd3522eb16346a1fc85f36ba4300fce51730a208a8022f24")
-    version("2.5.22", sha256="bd468d5b64909e3951342eacef30440e1eaca5d32add991fe6cc36b46f1c5f65")
-    version("2.5.17", sha256="5b9bd0e8e6359edb594416847a946214376f013afedebbd861ebd6556d45823f")
-    version("2.5.16", sha256="2de3a199407a378ea62f76652bc0945065d6636c7f7a02e5fafa9667f4ae833a")
+    homepage = "https://github.com/fnal-fife/nucondb"
+    git_base = "https://github.com/fnal-fife/nucondb.git"
```

Comment:
Does Spack understand `git_base`?  We don't see it anywhere in `fnal-develop`.  Perhaps it's just `git`?

Reply:
Several of our recipes used this historically, just as a class variable; for example:
dunetpc:
```
   git_base = "https://cdcvs.fnal.gov/projects/dunetpc"
...
   version("develop", branch="develop", git=git_base, get_full_repo=True)
```
 Thus you specify the git=git_base and don't repeat the string. 
 If we're no longer using it in any versions, we could certainly remove it or rename it.

Reply:
I'm not sure when it was introduced, but the class variable `git` performs this function: if that's what you want, you can specify (e.g.) `branch="develop"` without specifying `git=` at all, and the class-level value of `git` will be used.

Reply:
I would suggest a separate PR changing `git_base` to `git`, and removing now-redundant `git=git_base` clauses across the board in our recipe repos.